settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     repositories {
-        jcenter()
+        mavenCentral()
```

Comment:
Is there any code that requires this (i.e. did you check)? E.g. on sodium, no code required it, so they yeeted jcenter without replacing it, just a heads up. This is less of a change request and more a reminder.

Reply:
It isn't required so i will just remove it completely

Reply:
done